### PR TITLE
feat: add faq section

### PR DESCRIPTION
**Pull request by RickDeb2004**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Lucif3r-in** [comment]
Make the width full, animations are not according to figma file, follow figma properly

**Comment by RickDeb2004** [comment]
check this ..still not able to find why its going down the page....I have done min-height: 100 vh to gain the full view-port height

**Comment by SirSimon162** [comment]
@RickDeb2004 
- Use proper typography and align them as it is in the design
- Why is the FAQ section going down the page? Fix that
- Fix the mobile view as per the design
- Refer this article: https://cbea.ms/git-commit/

File: src/components/Faq/Faq.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    },
+    {
+      question: 'How do I register for the hackathon?',
+      answer: 'Answer to question 2..',
+    },
+    {
+      question: 'How do I register for the hackathon?',
+      answer: 'Answer to question 3.',
+    },
+  ];
```
Comment (Lucif3r-in) [inline src/components/Faq/Faq.jsx:38]: Keep these in config/content

**Comment by SirSimon162** [comment]
> check this ..still not able to find why its going down the page....I have done min-height: 100 vh to gain the full view-port height
Remove absolute positioning and remove height from the FAQContainer and add margin auto. Try doing this

File: src/components/Faq/Style.jsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    opacity: 1;
+    transform: rotate(0deg);
+  }
+`;
+
+export const FAQContainer = styled.div`
+  width: 1162px;
+  height: 1400px;
+  position: absolute;
+  top: 6109px;
```
Comment (Lucif3r-in) [inline src/components/Faq/Style.jsx:61]: What's this for?

**Review by Lucif3r-in — CHANGES_REQUESTED** [review]

**Comment by RickDeb2004** [comment]
> @RickDeb2004
> 
> * Use proper typography and align them as it is in the design
> * Why is the FAQ section going down the page? Fix that
> * Fix the mobile view as per the design
> * Refer this article: https://cbea.ms/git-commit/

I have replicated the typography as the figma has
faq section going down beacause I have added a faq container if I remove that then It will be fixed

**Comment by RickDeb2004** [comment]
> justify-content: center

yup..done

File: src/components/Faq/Faq.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  //     {
+  //       question: 'How do I register for the hackathon?',
+  //       answer: 'Answer to question 2..',
+  //     },
+  //     {
+  //       question: 'How do I register for the hackathon?',
+  //       answer: 'Answer to question 3.',
+  //     },
+  //   ];
+
```
Comment (Lucif3r-in) [inline src/components/Faq/Faq.jsx:40]: Remove this part

**Review by Lucif3r-in — CHANGES_REQUESTED** [review]

**Comment by Lucif3r-in** [comment]
image
There is an overflow @RickDeb2004

**Comment by RickDeb2004** [comment]
> image
> 
> * There is an overflow @RickDeb2004
> * Work on the animation it is still fading

I have not del that fading part

File: src/components/Faq/Style.jsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  border: 2px solid var(--neutral-3, #d9d9d9);
+  gap: 36px;
+  color: #fff;
+  font-family: Prompt;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 36px;
+  width: 1162px;
+  cursor: pointer;
+  white-space: nowrap;
```
Comment (SirSimon162) [inline src/components/Faq/Style.jsx:77]: Remove this line 
white-space: nowrap

**Comment by Lucif3r-in** [comment]
@RickDeb2004 Still fade animation is there when the page loads and after you close the accordion

File: src/components/Faq/Style.jsx (hunk 4 above)
Comment (RickDeb2004) [reply src/components/Faq/Style.jsx:77]: done ..thank you @SirSimon162

**Comment by Lucif3r-in** [comment]
Increase the transition time to 0.5s and the faq closing icon transition seems a little buggy, Can you fix that?

**Comment by RickDeb2004** [comment]
> Increase the transition time to 0.5s and the faq closing icon transition seems a little buggy, Can you fix that? @RickDeb2004

can you elaborate what to do with the closing icons ?

**Comment by Lucif3r-in** [comment]
> > Increase the transition time to 0.5s and the faq closing icon transition seems a little buggy, Can you fix that? @RickDeb2004
> 
> can you elaborate what to do with the closing icons ?

The icon is rotating after the accordion closes.

File: src/components/Faq/Faq.jsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  return (
+    <FAQContainer>
+      <Helmet>
+        <link
+          rel='stylesheet'
+          href='https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0'
+        />
+      </Helmet>
+      {/* <FAQHeading>Do You Have A Question?</FAQHeading> */}
```
Comment (ORKO06) [inline src/components/Faq/Faq.jsx:35]: Remove redundant comments

Comment (ORKO06) [inline src/components/Faq/Faq.jsx:35]: Remove redundant comments

File: src/components/Faq/Faq.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      <Heading2>Do You Have A Question?</Heading2>
+      {/* <FAQCaption>New to Hackathon! Dont worry</FAQCaption> */}
```
Comment (ORKO06) [inline src/components/Faq/Faq.jsx:37]: Remove redundant comments

File: src/components/Faq/Faq.jsx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+      <FAQCaption>
+        {' '}
```
Comment (ORKO06) [inline src/components/Faq/Faq.jsx:39]: Why is this required?

File: src/components/Faq/Style.jsx [hunk 8]
```diff
@@ -0,0 +1,124 @@
+import styled, { keyframes } from 'styled-components';
+import { Caption } from '../shared';
+
+const rotateOut = keyframes`
+  from {
+
```
Comment (ORKO06) [inline src/components/Faq/Style.jsx:6]: Remove unnecessary spacing

File: src/components/Faq/Style.jsx [hunk 9]
```diff
@@ -0,0 +1,124 @@
+import styled, { keyframes } from 'styled-components';
+import { Caption } from '../shared';
+
+const rotateOut = keyframes`
+  from {
+    
+    transform: rotate(0deg);
+  }
+  to {
+
```
Comment (ORKO06) [inline src/components/Faq/Style.jsx:10]: Remove unnecessary spacing

**Review by ORKO06 — CHANGES_REQUESTED** [review]

File: src/components/Faq/Faq.jsx [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+  };
+
+  return (
+    <FAQContainer>
+      <Helmet>
+        <link
+          rel='stylesheet'
+          href='https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0'
+        />
+      </Helmet>
```
Comment (Lucif3r-in) [inline src/components/Faq/Faq.jsx:34]: Import this at `global.css`

File: src/components/Faq/Style.jsx [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+  font-weight: 600;
+  line-height: 64px;
+  color: #fff;
+  margin-bottom: -35px;
+  @media (max-width: 768px) {
+    font-size: 24px;
+  }
+`;
+
+export const FAQCaption = styled.div``;
```
Comment (Lucif3r-in) [inline src/components/Faq/Style.jsx:45]: remove this if unused

File: src/components/Faq/Style.jsx [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+  @media (max-width: 768px) {
+    font-size: 16px;
+  }
+`;
+
+export const FAQIcon = styled.div`
+  //   width: 24px;
+  //   height: 24px;
+  position: absolute;
+  //   left: 12px;
```
Comment (Lucif3r-in) [inline src/components/Faq/Style.jsx:94]: remove unnecessary comments

File: src/components/Faq/Faq.jsx (hunk 7 above)
Comment (RickDeb2004) [reply src/components/Faq/Faq.jsx]: removed
@ORKO06

File: src/components/Faq/Style.jsx [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+  &.closing {
+    animation: ${rotateOut} 0.5s ease-in-out;
+  }
+  &.opened {
+    transform: rotate(0deg);
+  }
+`;
+
+export const FAQAnswer = styled(Caption)`
+  display: ${(props) => (props.isOpen ? 'block' : 'none')};
```
Comment (Shurtu-gal) [inline src/components/Faq/Style.jsx:104]: Put transition using max-height here for good animation.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/Faq/Style.jsx (hunk 13 above)
Comment (RickDeb2004) [reply src/components/Faq/Style.jsx]: okay

File: src/components/Faq/Style.js [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+    background: rgba(0, 0, 255, 0.1);
+  }
+
+  @media (max-width: 768px) {
+    font-size: 16px;
+    border: 12px;
+  }
+`;
+
+export const FAQIcon = styled.div`
```
Comment (SirSimon162) [inline src/components/Faq/Style.js:91]: For smaller screens this is moving inwards towards the question and for larger screens this is going out of the parent div. Try using a flex with justify-between.

File: src/components/Faq/styles.js [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+    border: 2px solid var(--neutral-3, #d9d9d9);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    padding: 16px 24px;
+  }
+`;
+export const FAQIcon = styled.div`
+  tarnsform: translateY(-50%);
```
Comment (Shurtu-gal) [inline src/components/Faq/styles.js:94]: Spelling 😲

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

**Comment by Shurtu-gal** [comment]
nit: @RickDeb2004 Try to use proper commit names like `fix:` instead of `BugFix:`

**Comment by Lucif3r-in** [comment]
LGTM🚀
